rir/src/compiler/pir/type.cpp

```diff
@@ -75,16 +80,14 @@ void PirType::merge(SEXPTYPE sexptype) {
     case WEAKREFSXP:
     case S4SXP:
         t_.r = val().t_.r;
+        break;
+    default:
+        break;
     }
 }
 
 PirType::PirType(SEXP e) : flags_(defaultRTypeFlags()), t_(RTypeSet()) {
-    if (e == R_MissingArg)
```

Comment:
why did you remove this. it is definetly needed. don't remove it!! we have special types for missing and unbound! they are typed as symbol in pir.